tests-integration/token_management_test.go

```diff
@@ -106,7 +106,85 @@ func TestSyncWithNewTokenAfterOldExpires(t *testing.T) {
 	m.MatchResponse(t, res,
 		m.MatchRoomSubscription(roomID, m.MatchRoomTimelineMostRecent(1, []json.RawMessage{bobMsg})),
 	)
+}
+
+func TestSyncWithNewTokenIncludesPollerUpdatesFromOldToken(t *testing.T) {
```

Comment:
This is https://github.com/matrix-org/sliding-sync/pull/106#discussion_r1194746629